code/modules/mob/living/carbon/human/say.dm

```diff
@@ -120,6 +120,31 @@
 /mob/living/carbon/human/proc/GetSpecialVoice()
 	return special_voice
 
+GLOBAL_LIST_INIT(soapy_words, list(
+	"shit" = "snap",
+	"shitter" = "toilet",
+	"fuck" = "phooey",
+	"fucking" = "flipping",
+	"fucker" = "individual dedicated to the continuation of their species",
+	"motherfucker" = "family time enjoyer",
+	"crap" = "crud",
```

Comment:
I don't think this one should be included.

Reply:
Agree, the UK broadcasting standards rates `crap` as: 
![image](https://user-images.githubusercontent.com/12197162/212423222-26d0f78d-d0d8-416c-bd55-fa6b651c05a9.png)

Suggest you have a look at this list here to identify a range of widely accepted offensive words: https://www.ofcom.org.uk/__data/assets/pdf_file/0020/225335/offensive-language-quick-reference-guide.pdf

Reply:
Yup, crap only made the list because I asked the fellas on discord if they thought I missed any.